minidrivycontrol.rb

```diff
@@ -48,11 +48,30 @@ def initialize(id, car_id, days, kms)
 
 end
 
+class Car
+
+  def initialize(id, price_per_day, price_per_km)
+    @id = id
+    @price_per_day = price_per_day
+    @price_per_km = price_per_km
+    @booked = false
+  end
+
+  def booking
```

Comment:
since this method modifies the object, it would be better to use a verb rather than a noun